src/components/Logo/Logo.css

```diff
@@ -0,0 +1,6 @@
+.Logo {
+    @apply
+        max-w-sm
```

Comment:
need to add `relative` so that the absolutely positioned text-screen-reader will stay within the div and hopefully not cause display issues when being used in different contexts